flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/main/java/org/apache/flink/cdc/connectors/mysql/source/config/MySqlSourceConfig.java

```diff
@@ -132,6 +133,18 @@ public class MySqlSourceConfig implements Serializable {
         this.dbzProperties = checkNotNull(dbzProperties);
         this.dbzConfiguration = Configuration.from(dbzProperties);
         this.dbzMySqlConfig = new MySqlConnectorConfig(dbzConfiguration);
+        Selectors excludeTableFilter =
+                (excludeTableList == null
+                        ? null
+                        : new Selectors.SelectorsBuilder().includeTables(excludeTableList).build());
+        Tables.TableFilter tableFilter = dbzMySqlConfig.getTableFilters().dataCollectionFilter();
+        dbzMySqlConfig
+                .getTableFilters()
+                .setDataCollectionFilters(
+                        (TableId tableId) ->
+                                tableFilter.isIncluded(tableId)
+                                        && (excludeTableFilter == null
+                                                || !excludeTableFilter.isMatch(tableId)));
```

Comment:
In mysql cdc connector, we need to disable to pass 'debezium.table.include.list' and 'debezium.table.exclude.list'.
Please add some check and warning for these options.

Reply:
Done in DebeziumOptions.